templates/network/resources/vkcs_networking_subnet.md.tmpl

```diff
@@ -12,6 +12,9 @@ description: |-
 
 ## Example Usage
 {{tffile .ExampleFile}}
+
+## use with custom dns
```

Comment:
Subnet with not default DNS servers

Reply:
do not see this change in than commit

Reply:
i do not know why github not showing it is in here 
https://github.com/vk-cs/terraform-provider-vkcs/pull/316/commits/9729597ceedfac727b6dbafa62218891534e6d35